dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/exchange/support/header/HeaderExchangeHandler.java

```diff
@@ -209,10 +209,16 @@ public void received(Channel channel, Object message) throws RemotingException {
                     Exception e = new Exception("Dubbo client can not supported string message: " + message + " in channel: " + channel + ", url: " + channel.getUrl());
                     logger.error(e.getMessage(), e);
                 } else {
-                    String echo = handler.telnet(channel, (String) message);
-                    if (echo != null && echo.length() > 0) {
-                        channel.send(echo);
+                    boolean enableTelnet = channel.getUrl().getParameter(Constants.TELNET, true);

```

Comment:
Constants.TELNET value and config value which user will provide is "enable.telnet" and in code we are using the variable name 'enableTelnet' which more aligned with value (eventually which user will be providing). Should we consider renaming the constant name to the same e.g. 
Constants.ENABLE_TELNET="enable.telnet" ?

Another thought, should we refactor the this method instead of directly using 'channel.getUrl().getParameter(Constants.TELNET, true);' can create a method named isTenletEnabled()

private Boolean isTenletEnabled(Channel channel) {
   return channel.getUrl().getParameter(Constants.TELNET, true);
}

In this way I think in future even we change constant name of add more parameter to conclude telnet is configured not it will be a single place change. This is just an my thought I could be over thinking here 😄

Reply:
already rename the constant value to TELNET_ENABLED to keep it's name the same as ProtocolConfig#telnetEnabled
about to add a method like 
```
private Boolean isTenletEnabled(Channel channel) {
return channel.getUrl().getParameter(Constants.TELNET, true);
}
```
I think is unnecessary,Does anyone have a different opinion? We can discuss it.